Api/DTO/Comments/CommentDto.cs

```diff
@@ -5,6 +5,8 @@ namespace Api.DTO.Comments;
 public record CommentDto(
     Guid Guid,
     PublicUserDto? Author,
+    bool? UserReactionIsDislike,
+    int LikeCount,
```

Comment:
It is named LikeCount, but it is both likes and dislikes. I guess it should matter?

Reply:
I separated them now